index.php

```diff
@@ -13,7 +13,11 @@
         $script = [];
         $script['file'] = $file;
         if (preg_match('/.php$/i', $file)) {
-            $output = exec('php -f scripts/' . $file .' 2>&1');
+            $content = file_get_contents('scripts/' . $file, true);
```

Comment:
No. The instruction said we should use exec()...

Nice work @Douglasokolaa But I don't think we can use this

Reply:
Then we might need to archive this branch so as not to cause slow motions like that in movies (lol)